<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2006-2010 Felix Geyer</author>
 <class>AboutDlg</class>
 <widget class="QDialog" name="AboutDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About WebMonX</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelIcon">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelVersion">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAbout">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <widget class="QLabel" name="labelA1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>74</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>WebMonX</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelA3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>351</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Copyright (C) 2006-2010 Felix Geyer</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelA2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>341</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>based on WebMon by Colin Markwell</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelA5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>222</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>http://debfx.fobos.de/webmonx/</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelA4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>191</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>debfx-webmonx@fobos.de</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLibs">
      <attribute name="title">
       <string>Libraries</string>
      </attribute>
      <widget class="QLabel" name="labelLibs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>381</width>
         <height>171</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLicense">
      <attribute name="title">
       <string>License</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
